src/build/buildNativefierApp.ts

```diff
@@ -129,6 +129,10 @@ function trimUnprocessableOptions(options: AppOptions): void {
 export async function buildNativefierApp(
   rawOptions: RawOptions,
 ): Promise<string | undefined> {
+  if (rawOptions.quiet) {
+    log.setLevel('silent');
+  }
+
```

Comment:
Hmmmmm do you really have to do this? Grepping our codebase I think you need to adjust:

1. `optionsMain.ts` (which you did)
2. `cli.ts` → Can you remove this change in `buildNativefierApp.ts` and instead adjust the `setLevel` in `cli.ts` ?

Reply:
So I actually do have a bit of a justification for this change – originally I just kept my changes inside `optionsMain.ts` but then realized that they would only take affect after calling `getOptions()`, and unfortunately there are log statements in this file that would execute before I had a chance to call `getOptions()` (on line 164). I tried moving the options initialization, but as this function mutates the data passed into `getOptions()`, moving it caused the function's logic to break. 

I would put it inside `cli.ts`, except that the purpose of this change (at least for me) was to be able to silence logger output when using the programmatic API, which doesn't rely on `cli.ts` AFAIK. I'm importing `buildNativefierApp` directly. The only other solution would be to have the change in `cli.ts` but also export the `log` variable so that my code (or other programmatic uses) could manually adjust the log level before calling `buildNativefierApp()`. 

Does this justification make sense?

Reply:
@Nickersoft 👍 that makes total sense. Adding a lil' comment summing this up above `if (rawOptions.quiet) {`

The rest of the PR looks gud. Merging, thanks! Will ship in the next release, in a few days/weeks.